rna/rna-alignment/README.md

```diff
@@ -0,0 +1,87 @@
+# RNAAlignment with STAR
+
+This workflow runs the STAR command on a pair of fastq files.
+(https://github.com/alexdobin/STAR) and produces an aligned, unsorted BAM
+with STAR's default compression level (1).
+
+This output BAM for this workflow can be sorted, compressed, and indexed
+with the `RNABAMSortAndIndex` workflow.
+
+Inputs:
+- Per-sample:
+  - Sample name
+  - R1 FASTQ file list
+  - R2 FASTQ file list
+
+- Reference:
+  - STAR index directory tar file (.tar.gz)
+
+- VM configuration
+  - docker image url
+  - VM disk size
+  - VM memory
+  - Runtime zones, ex: "us-central1-a us-central1-b"
+  - Number of times to try the workflow with a preemptible VM before
+    falling back to a full-price VM.
+
+Outputs:
+- Per-sample
+  - &lt;sample-id&gt;.Aligned.out.bam
+  - &lt;sample-id&gt;.SJ.out.tab
+  - &lt;sample-id&gt;.Chimeric.out.junction
+  - &lt;sample-id&gt;.Log.final.out
+  - &lt;sample-id&gt;.Log.out
+  - &lt;sample-id&gt;.Log.progress.out
+    - &lt;sample-id&gt;.STARgenome
+      - sjdbInfo.txt
+      - sjdbList.out.tab
+    - &lt;sample-id&gt;.STARpass1
+      - Log.final.out
+      - SJ.out.tab
+
+## Notes
+A sample inputs.json file is included here with values derived from running workflows for AMP PD on [Terra](https://app.terra.bio/).
+
+By default, the workflow will run on preemptible machines, but will hard-terminate the STAR command at 23.5
+hours. This is to avoid getting preempted at 24 hours and then re-running on a preemptible VM, which again
+won't succeed. After hard termination, the long-running sample can be restarted explicitly on a non-preemptible VM.
+
+To disable preemptible machines for long running samples, set `RNAAlignment.preemptible_tries` to 0 and `RNAAlignment.star_timeout_hours` to a very large value.
+
+See [broadinstitute/cromwell#4600](https://github.com/broadinstitute/cromwell/issues/4600)
```

Comment:
Please switch the order of these two paragraphs "See..." and "The timeout is applied..."